mlflow/protos/databricks_artifacts.proto

```diff
@@ -155,3 +192,75 @@ message GetCredentialsForWrite {
 
   }
 }
+
+message CreateMultipartUpload {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+  option (scalapb.message).extends = "com.databricks.mlflow.api.MlflowTrackingMessage";
+
+  // Run ID
+  optional string run_id = 1 [(validate_required) = true];
+
+  // Artifact paths, relative to the Run's artifact root location
+  optional string path = 2;
+
+  // Number of parts to upload in the initiated multipart upload
+  optional int64 num_parts = 3;
+
+  message Response {
+    // ID identifying the initiated multipart upload
+    optional string upload_id = 4;
+
+    // Credentials for uploading parts in the initiated multipart upload
+    repeated ArtifactCredentialInfo upload_credential_infos = 5;
+
+    // Credential for aborting the initiated multipart upload
+    optional ArtifactCredentialInfo abort_credential_info = 7;
```

Comment:
Response fields should restart at 1, so this should be 1, 2, 3 instead of 4, 5, 7

Reply:
Can we make this change for other response messages as well?

Reply:
Fixed!